.travis.yml

```diff
@@ -20,5 +22,9 @@ after_success:
     test "${TRAVIS_PULL_REQUEST}" = 'false' &&
     test "${TRAVIS_REPO_SLUG}" = 'radicalbit/NSDb' &&
     test "${TRAVIS_JDK_VERSION}" = 'oraclejdk8' &&
-    sh "$TRAVIS_BUILD_DIR/travis-credentials.sh" &&
-    sbt +publish
+    sh "$TRAVIS_BUILD_DIR/travis-credentials.sh"
+  - if [[ "$TRAVIS_BRANCH" == "master" || "$TRAVIS_BRANCH" == branch-* && "$TRAVIS_EVENT_TYPE" == "push" ]] || [[ "$TRAVIS_TAG" != "" ]]  && [[ "$TRAVIS_REPO_SLUG" == "radicalbit/NSDb" ]]; then
+      sbt publish ;
+      docker login -u $DOCKER_USER -p $DOCKER_PASSWORD ;
+      sbt docker:publish ;
+    fi
```

Comment:
IMHO on `script` section should be invoked also an `sbt docker:publishLocal` to be sure that at least on every push also docker image creation works

Reply:
Added the command before the `sbt docker:publish ;`